module/TaskManagement/src/TaskManagement/Service/EventSourcingTaskService.php

```diff
@@ -107,4 +178,86 @@ public function findAcceptedTasksBefore(\DateInterval $interval){
 		
 		return $query->getResult();
 	}
+
+	public function getStatsForMember(Organization $org, $memberId, $queryOptions){
+
+		$countTasksOwner = $this->countTasksOwner($org, $memberId, $queryOptions);
+		$taskMemberInClosedTasks = $this->findTaskMemberInClosedTasks($org, $memberId, $queryOptions);
+		$sumTasksCredits = 0;
+		$averageOfDeltaShares = null;
+		if(!empty($taskMemberInClosedTasks)){
+			$countDeltaShares = 0;
+			$sumOfDeltaShares = 0;
+			foreach ($taskMemberInClosedTasks as $member){
+				$sumTasksCredits += $member->getCredits();
+				if(!is_null($member->getDelta())){
+					$sumOfDeltaShares += $member->getDelta();
+					$countDeltaShares++;
+				}
+			}
+			$averageOfDeltaShares = $countDeltaShares > 0 ? $sumOfDeltaShares / $countDeltaShares : null;
+		}
+		return[
+				"countTasksOwner" => $countTasksOwner,

```

Comment:
Dovendo essere dei sostantivi, una forma plausibile in inglese è "ownershipsCount", "creditsCount" e "averageDelta"